packages/ember-routing/lib/system/route.js

```diff
@@ -1616,7 +1616,9 @@ var Route = EmberObject.extend(ActionHandler, Evented, {
     needs access to a parent route's model in order to
     resolve a model (or just reuse the model from a parent),
     it can call `this.modelFor(theNameOfParentRoute)` to
-    retrieve it.
+    retrieve it. Returns an object, not a promise, 
```

Comment:
Otherwise I'm not sure it adds useful info for the reader? I doubt I'm being objective as I had a particular problem here with promises.